src/components/TagModal.vue

```diff
@@ -155,6 +177,34 @@ export default {
 				return `rgba(${r}, ${g}, ${b}, ${opacity})`
 			}
 		},
+		openEditTag() {
+			this.renameTagLabel = false
+			this.renameTagInput = true
+			this.showSaving = false
+
+		},
+		async renameTag(tag, event) {
+			this.renameTagInput = false
+			this.showSaving = false
+			const displayName = event.target.querySelector('input[type=text]').value
+
+			try {
+				await this.$store.dispatch('updateTag', {
+					tag,
+					displayName,
+					color: tag.color,
+				})
+				this.renameTagLabel = true
+				this.renameTagInput = false
+				this.showSaving = false
+			} catch (error) {
+				showInfo(t('mail', 'An error occurred, unable to rename the tag.'))
+				console.error(error)
+				this.renameTagLabel = false
+				this.renameTagInput = false
+				this.showSaving = true
```

Comment:
i set this to true, because when we have an error( that popups on top-right of the screen), the saving message " Saving tag …" will be shown and the user will not be able to click save more than once if they miss the popup error.  So this is true only if we have an error